appcat.yaml

```diff
@@ -1,7 +1,7 @@
 --- !<Application>
 id: "077a600f-29ce-443a-92b4-6d8656ae11ff"
 modelType: "Application"
-appCode: "eventhub"
```

Comment:
@slee1016  Previously, in the nakadi POC time, we used the appCode as nakadi for this repo. Then after POC ended and started eventhub we changed the appCode from nakadi to eventhub. So shall we revise the appCode as event-hub now.

Reply:
Ok @manusha-pieris , I just changed the appCode to event-hub if you can confirm the change looks good

Reply:
Nvm just spoke to @jojijohn -- the appCode will be nakadi